easybuild/easyconfigs/p/PyTables/PyTables-3.8.0-foss-2023a.eb

```diff
@@ -47,11 +47,11 @@ exts_list = [
     }),
     ('tables', version, {
         'patches': [
-            'PyTables-%(version)s_fix-libs.patch',
+            'PyTables-3.8.0_fix-libs.patch',
             'PyTables-3.8.0_fix-find-blosc2-library-path.patch',
         ],
         'checksums': [
-            {'tables-3.8.0.tar.gz': '34f3fa2366ce20b18f1df573a77c1d27306ce1f2a41d9f9eff621b5192ea8788'},
+            {'tables-%(version)s.tar.gz': '34f3fa2366ce20b18f1df573a77c1d27306ce1f2a41d9f9eff621b5192ea8788'},
```

Comment:
These 2 changes looks random, was this intended?

Reply:
Hmm, sure what happened here, maybe there was a merge upstream of a version with only some of these diffs. Don't see any reason not to delete PyTables changes entirely